pkgs/applications/kde/kdevelop/kdev-python.nix

```diff
@@ -1,13 +1,13 @@
-{ stdenv, lib, fetchurl, cmake, extra-cmake-modules, threadweaver, ktexteditor, kdevelop-unwrapped, python }:
-
-stdenv.mkDerivation rec {
+{ mkDerivation, lib, cmake, extra-cmake-modules
+, threadweaver, ktexteditor, kdevelop-unwrapped, python39
+}:
+let
+  # FIXME: stick with py39 until
```

Comment:
```suggestion
  # FIXME: use python 3.9 until 3.10 has less issues
```

Reply:
This is not so much about py3.10 itself, but instead about kdev-python not yet having being updated; open MR url right below, just broke the line to avoid it being too long.

Reply:
Adjusted comment to explain this better